addOns/network/src/test/java/org/zaproxy/addon/network/GlobalExclusionsOptionsUnitTest.java

```diff
@@ -215,6 +218,33 @@ void shouldMigrateCoreOptions() {
         assertExclusion(0, "Name", "Value", false);
     }
 
+    @Test
+    void defaultExclusionsShouldAllBeCsaeInsensitive() {
```

Comment:
There's a typo in Case, though better follow the existing convention, `shouldHaveAllDefaultExclusionsCaseInsensitive`.